.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,32 @@
+name: Chorus .NET6
+
+on:
+  [push, pull_request, workflow_dispatch]
+
+jobs:
+  build_test_deploy:
+    env:
+      NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Why do we need this token?

Reply:
It's required during .NET setup. i removed it to verify, and things break